pyracing/client.py

```diff
@@ -42,6 +42,31 @@ def __init__(self, username: str, password: str):
         self.password = password
         self.session = httpx.AsyncClient()
 
+    def _rename_numerical_keys(self, response_item, mapping):
+        """ Returns a version of 'response_item' where all numerical keys have 
+        been renamed to readable strings by applying 'mapping'.
+        """
+        if not isinstance(response_item, dict):
+            logger.warning('The response_item you provided does not appear to be a '
```

Comment:
I think generally we want the logger message to be more for the user than this. The user using this won't be calling rename numerical keys, theyll be calling one of our public methods. You could maybe return an error message containing the mapping or the response, or telling them that the method they called has changed and doesn't respond well anymore

Reply:
Logger message changed and checks combined to one line.